projects/packages/stats/src/class-tracking-pixel.php

```diff
@@ -107,7 +107,11 @@ public static function enqueue_stats_script() {
 		);
 
 		// Make sure the script loads asynchronously (add a defer attribute).
-		Assets::instance()->add_async_script( 'jetpack-stats' );
+		if ( version_compare( $wp_version, '6.3', '>=' ) ) {
```

Comment:
May as well just update the `wp_enqueue_script()` call above. We're in the process of dropping pre-6.3 code paths.